src/lambda_toolshed/papillon.cljc

```diff
@@ -159,10 +155,21 @@
      by the chain)."
   ([ixs]
    (execute ixs {}))
-  ([ixs ctx]
+  #?(:org.babashka/nbb nil
```

Comment:
Don't give the 2-arity to `nbb` as `nbb` doesn't support `core.async`

CLJ and CLJS get the nice behavior we see in the tests about being able to get async results on a channel.